docs/platforms/apple/guides/ios/session-replay/performance-overhead.mdx

```diff
@@ -61,20 +61,15 @@ SentrySDK.start(configureOptions: { options in
 })
 ```
 
-### Enable Experimental View Renderer
-
-<Alert>
-  In case you are noticing issues with the experimental view renderer, please
-  report the issue on [GitHub](https://github.com/getsentry/sentry-cocoa).
-</Alert>
+### Disable View Renderer V2
 
 Starting with v8.47.0 you can enable the up-to-5x-faster new view renderer, reducing the impact of Session Replay on the main thread and potential frame drops.
+As of v8.50.0, the new view renderer is enabled by default.
 
-While we do recommend the new view renderer, it's currently considered an experimental feature to further evaluate its stability.
-After the evaluation phase we are going to enable it by default.
+While we do recommend the new view renderer, if you are experiencing issues, you can disable it by setting the following flag:
```

Comment:
```suggestion
While we do recommend the new view renderer, if you are experiencing issues, you can opt out of using it by setting the following flag:
```

Reply:
`m`: also add a note that at a later point the option will likely be deprecated and removed at some point (e.g. the next major)

Reply:
I added a note